public/js/main.js

```diff
@@ -68,7 +74,7 @@ function hasSelectedTab() {
  */
 function getSelectedTab(tabs) {
   const childId = window.location.hash.split("=")[1];
-  return tabs.find(tab => tab.get("actor").includes(childId));
+  return tabs.find(tab => tab.get("id").includes(childId));
```

Comment:
Switching from `actor` to `id` is a good idea, we'll probably do that for all objects